Misc/NEWS.d/next/Library/2018-09-26-15-18-18.bpo-34800.auD5YK.rst

```diff
@@ -0,0 +1,2 @@
+Fix `email.contentmanager.set_content` raise error when
```

Comment:
Some small suggestions:

```
Fixed a bug where ``email.contentmanager.set_content`` raised error when
``email.policy.Policy.max_line_length`` is 0 or ``None``. Patch by silane.
```

In rst, `<code>` sections needs to be wrapped in double backticks.